CanHazFunny/CanHazFunny.Tests/JokeToScreenTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Moq;
+using System.Globalization;
+using System.IO;
+
+namespace CanHazFunny.Tests;
+
+public class JokeToScreenTests
+{
+    [Fact]
+    public void PrintJoke_OutputsJoke_Successful()
+    {
+        JokeToScreen jokeToScreen = new();
+        StringWriter stringWriter = new(CultureInfo.InvariantCulture);
+        Console.SetOut(stringWriter);
+        jokeToScreen.PrintJoke("Test Joke");
+        string consoleOutput = stringWriter!.ToString();
+
+        Assert.Equal("Test Joke" + Environment.NewLine, consoleOutput);
```

Comment:
```suggestion
        Assert.NotNull(stringWriter);
        Assert.Equal("Test Joke" + Environment.NewLine, stringWriter.ToString());
```
This is simpler and better checks for `NotNull`